### ISSUE TITLE
Request to release 0.15

### ISSUE THREAD

**Issue by rafaelcioban** [taskadapter/trello-java-wrapper#88]
Hello folks 👋🏻
I am currently stumbling on the following error when invoking the `getListCards()` method:
```
com.fasterxml.jackson.databind.JsonMappingException: Numeric value (663355555077.0367) out of range of int (-2147483648 - 2147483647)
 at [Source: (String)"[{"id":"65369888efaa0e8bc170b036","badges":{"attachmentsByType":{"trello":{"board":0,"card":0}},"location":false,"votes":0,"viewingMemberVoted":false,"subscribed":false,"fogbugz":"","checkItems":0,"checkItemsChecked":0,"checkItemsEarliestDue":null,"comments":10,"attachments":0,"description":true,"due":"2023-11-09T16:00:00.000Z","dueComplete":true,"start":null},"checkItemStates":[],"closed":false,"dueComplete":true,"dateLastActivity":"2023-11-08T19:15:58.919Z","desc":"### Sua tarefa é sobre:\n\nO"[truncated 5817 chars]; line: 1, column: 3308] (through reference chain: java.lang.Object[][1]->com.julienvey.trello.domain.Card["pos"])
```
And I see that the following fix has already been provided for this: [Changing pos from int to long](https://github.com/taskadapter/trello-java-wrapper/commit/68df1caca2c40b1881e0571039bfad5dd7714b25) (https://github.com/taskadapter/trello-java-wrapper/pull/76).
Are there any plans to release version `0.15` with this fix? Happy to help if needed. 
Thanks,
Rafael

**Comment by proteus1121** [comment]
Hi, @rafaelcioban 
Not sure that it's still relevant to you, but it's looks like this repo was abandon, so I create a fork and published 0.15 version.
You can find it there: https://central.sonatype.com/artifact/io.github.proteus1121/trello-java-wrapper

**Comment by rafaelcioban** [comment]
Hi @proteus1121 ,
Thanks for coming back to me on this. It's still relevant indeed, thanks for forking & publishing the 0.15 version! Just to make sure: this is meant to remain published permanently, right? Just asking as I plan to add such dependency to my project. Cheers!

**Comment by proteus1121** [comment]
@rafaelcioban Yep, it was published permanently and I'm going to support this as much as I can. So if you need any updates\modifications feel free to open issues\pull requests to [forked repo](https://github.com/proteus1121/trello-java-wrapper) and I'll publish updates.
